modules/javafx.graphics/src/main/java/javafx/scene/layout/HeaderBar.java

```diff
@@ -65,23 +67,23 @@
  * method.
  * <p>
  * {@code HeaderBar} is a layout container that allows applications to place scene graph nodes in three areas:
- * {@link #leadingProperty() leading}, {@link #centerProperty() center}, and {@link #trailingProperty() trailing}.
+ * {@link #leftProperty() left}, {@link #centerProperty() center}, and {@link #rightProperty() right}.
```

Comment:
perhaps there should be a separate section explaining that the left/right nodes get switched visually in the RTL mode?  And/or mention that "left" really means "leading", and "right" really means "trailing"?